lib/StaticNarrative/exporter/objectswithsets.py

```diff
@@ -0,0 +1,134 @@
+import json
+import os
+from typing import Any
```

Comment:
a few minor edits in this file:
- remove unused input variables
- add in some code for debugging (the json dump code that's commented out)